### ISSUE TITLE
FEATURE REQUEST - DCA Buy at set negative Profit level value disregarding trailing

### ISSUE THREAD

**Issue by jbhitman** [taniman/profit-trailer#902]
Basically need to DCA at each level, at the set -percentage value, disregarding trailing.  LOSS buy strategy in essence, but instead of the 24H change, it needs to use the percentage change of the Pair in DCA relative to your buy in value.

**Comment by keithofaptos** [comment]
yes

**Comment by a7777777v** [comment]
cant you do this with trailing = 0 ?

**Comment by YoJeff** [comment]
This sounds exactly like ANDERSON
https://wiki.profittrailer.io/doku.php/start

**Comment by jbhitman** [comment]
Yes...I'm fully aware of how Anderson DCA works...Perhaps I should have made my request in simpler terms so you could actually understood what I was asking, but it doesn't matter..spoke with Elroy and he personally clarified how to do what I wanted.  Thanks for taking 0 time trying to understand the question.
